.github/workflows/spelling2.yml

```diff
@@ -91,88 +91,97 @@ jobs:
-        experimental_apply_changes_via_bot: ${{ github.repository_owner != 'microsoft' && 1 }}
-        use_sarif: ${{ (!github.event.pull_request || (github.event.pull_request.head.repo.full_name == github.repository)) && 1 }}
-        extra_dictionary_limit: 20
-        extra_dictionaries:
-          cspell:aws/aws.txt
-          cspell:cpp/src/compiler-clang-attributes.txt
-          cspell:cpp/src/compiler-msvc.txt
-          cspell:cpp/src/lang-keywords.txt
-          cspell:cpp/src/stdlib-c.txt
-          cspell:cpp/src/stdlib-cmath.txt
-          cspell:cpp/src/stdlib-cpp.txt
-          cspell:csharp/csharp.txt
-          cspell:css/dict/css.txt
-          cspell:django/dict/django.txt
-          cspell:dotnet/dict/dotnet.txt
-          cspell:filetypes/filetypes.txt
-          cspell:fullstack/dict/fullstack.txt
-          cspell:golang/dict/go.txt
-          cspell:html/dict/html.txt
-          cspell:java/src/java.txt
-          cspell:java/src/java-terms.txt
-          cspell:k8s/dict/k8s.txt
-          cspell:lorem-ipsum/dictionary.txt
-          cspell:monkeyc/src/monkeyc_keywords.txt
-          cspell:node/dict/node.txt
-          cspell:php/dict/php.txt
-          cspell:powershell/dict/powershell.txt
-          cspell:python/src/common/extra.txt
-          cspell:python/src/python/python.txt
-          cspell:python/src/python/python-lib.txt
-          cspell:scala/dict/scala.txt
-          cspell:software-terms/dict/softwareTerms.txt
-          cspell:swift/src/swift.txt
-          cspell:typescript/dict/typescript.txt
+      - name: check-spelling
+        id: spelling
+        uses: check-spelling/check-spelling@v0.0.24
+        with:
+          config: .github/actions/spell-check
+          suppress_push_for_open_pull_request: ${{ github.actor != 'dependabot[bot]' && 1 }}
+          checkout: true
+          check_file_names: 1
+          spell_check_this: microsoft/PowerToys@main
+          post_comment: 0
+          use_magic_file: 1
+          report-timing: 1
+          warnings: bad-regex,binary-file,deprecated-feature,ignored-expect-variant,large-file,limited-references,no-newline-at-eof,noisy-file,non-alpha-in-dictionary,token-is-substring,unexpected-line-ending,whitespace-in-dictionary,minified-file,unsupported-configuration,no-files-to-check,unclosed-block-ignore-begin,unclosed-block-ignore-end
+          experimental_apply_changes_via_bot: 1
+          use_sarif: ${{ (!github.event.pull_request || (github.event.pull_request.head.repo.full_name == github.repository)) && 1 }}
+          check_extra_dictionaries: ""
+          dictionary_source_prefixes: >
+            {
+            "cspell": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20241114/dictionaries/"
```

Comment:
I intend to reorganize the dictionaries for the next version, so this set is based on the expected next version's list. The changes didn't seem particularly harmful to this repository (which can be measured by additions to the `allow/` directory / entries in `expect.txt`).